tests/migration/Core/V6_4/Migration1665267882RenameCountryVatTest.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopware\Tests\Migration\Core\V6_4;
+
+use Doctrine\DBAL\Connection;
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Framework\Test\TestCaseBase\KernelLifecycleManager;
+use Shopware\Core\Migration\V6_4\Migration1665267882RenameCountryVat;
+use Shopware\Tests\Migration\MigrationTestTrait;
+
+class Migration1665267882RenameCountryVatTest extends TestCase
+{
+    use MigrationTestTrait;
+
+    private const NAME_EN = 'Holy See';
+    private const NAME_DE = 'Heiliger Stuhl';
+
+    private Connection $connection;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->connection = KernelLifecycleManager::getConnection();
+        $this->prepare();
+    }
+
+    public function testCountryVaticanCityExist(): void
+    {
+        $countryId = $this->connection->fetchOne('SELECT id FROM country WHERE iso = :iso AND iso3 = :iso3', ['iso' => 'VA', 'iso3' => 'VAT']);
+
+        static::assertIsNotBool($countryId, 'Country \'Vatican City\' not found');
+    }
+
+    public function testFetchOfNonExistingCountryId(): void
+    {
+        $countryId = $this->connection->fetchOne('SELECT id FROM country WHERE iso = :iso AND iso3 = :iso3', ['iso' => 'VA3', 'iso3' => 'VAT3']);
+
+        static::assertIsBool($countryId);
+        static::assertEquals($countryId, false);
```

Comment:
```suggestion
        static::assertFalse($countryId);
```
no need to perform two assertions

Reply:
Actually, I wanted to use the function, but Phpstorm couldn't find it at first.